From: Ahmad Fatoum <a.fatoum@pengutronix.de>
Date: Fri, 5 Jul 2024 10:39:04 +0200
Subject: [PATCH] ARM: skov-imx8mp: print initial > on every boot

putc_ll expands to nothing if CONFIG_DEBUG_LL is not defined. As we have
normal CONFIG_PBL_CONSOLE support, we can just use normal printing
functions to always output. Let's do that, so we have a very early
character (>) printed to console to indicate that barebox was entered,
even if further boot was to hang shortly after for whatever reason.

Signed-off-by: Ahmad Fatoum <a.fatoum@pengutronix.de>
---
 arch/arm/boards/skov-imx8mp/lowlevel.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boards/skov-imx8mp/lowlevel.c b/arch/arm/boards/skov-imx8mp/lowlevel.c
index 4338d511c4c2..d84fd73261ca 100644
--- a/arch/arm/boards/skov-imx8mp/lowlevel.c
+++ b/arch/arm/boards/skov-imx8mp/lowlevel.c
@@ -45,7 +45,7 @@ static void setup_uart(void)
 
 	pbl_set_putc(imx_uart_putc, uart);
 
-	putc_ll('>');
+	putchar('>');
 }
 
 static struct pmic_config pca9450_cfg[] = {
